examples/post_training_quantization/torch/mobilenet_v2/README.md

```diff
@@ -0,0 +1,26 @@
+# Post-Training Quantization of MobileNet v2 PyTorch Model
+This sample demonstrates how to use Post-Training Quantization API from Neural Network Compression Framework (NNCF) to quantize PyTorch models on the example of [MobileNet v2](https://huggingface.co/AlexKoff88/mobilenet_v2_food101)quatization, pretrained on [Food101](https://pytorch.org/vision/stable/generated/torchvision.datasets.Food101.html) dataset.
+
+
+The sample includes the following steps:
+- Loading the [Food101](https://pytorch.org/vision/stable/generated/torchvision.datasets.Food101.html) dataset (~4.7 Gb) and the [MobileNet v2 PyTorch model](https://huggingface.co/AlexKoff88/mobilenet_v2_food101) pretrained on this dataset.
+- Quantizing the model using NNCF Post-Training Quantization algorithm.
+- Output of the following characteristics of the quantized model:
+    - Accuracy drop of the quantized model (INT8) over the pre-trained model (FP32)
+    - Compression rate of the quantized model file size relative to the pre-trained model file size
+    - Performance speed up of the quantized model (INT8)
+
+# Install requirements
+At this point it is assumed that you have already installed nncf. You can find information on installation nncf [here](https://github.com/openvinotoolkit/nncf#user-content-installation).
+
+To work with the sample you should install the corresponding Python package dependencies:
+```
+pip install -r requirements.txt
+```
+
+# Run Sample
+It's pretty simple. The sample does not require additional preparation. It will do the preparation itself, such as loading the dataset and model, etc.
+> **NOTE**: If you have already downloaded the [Food101](https://pytorch.org/vision/stable/generated/torchvision.datasets.Food101.html) dataset to the disk, please provide the path to the daatset to avoid re-downloading.
```

Comment:
```suggestion
> **NOTE**: If you have already downloaded the [Food101](https://pytorch.org/vision/stable/generated/torchvision.datasets.Food101.html) dataset to the disk, please provide the path to the dataset to avoid re-downloading.
```